splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/SpliceAccessManager.java

```diff
@@ -326,6 +326,48 @@ public TransactionController getTransaction(ContextManager cm) throws StandardEx
         return getAndNameTransaction(cm, AccessFactoryGlobals.USER_TRANS_NAME);
     }
 
+    public TransactionController getReadOnlyTransaction(ContextManager cm, long id) throws StandardException {
+        String transName = AccessFactoryGlobals.USER_TRANS_NAME;
+
+        /*
+         * This call represents the top-level transactional access point. E.g., the top-level user transaction
+         * is created by a call to this method. In this case we are creating a read only transaction at a predefined point
+         */
+        if (LOG.isDebugEnabled())
+            LOG.debug("in SpliceAccessManager - getAndNameTransaction, transName=" + transName);
```

Comment:
Debug tracing message may be confusing. Either change it or refactor code to avoid copying code.